tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -90,25 +82,17 @@ var _ = Describe("odo devfile push command tests", func() {
 			Expect(output).NotTo(ContainSubstring(cmpName))
 
 			helper.RenameFile("devfile-old.yaml", "devfile.yaml")
-			output = helper.CmdShouldPass("odo", "push", "--project", namespace)
+			output = helper.CmdShouldPass("odo", "push", "--project", namespace, cmpName)
```

Comment:
What if we don't pass `cmpName` with `odo push`. Does it mean that won't work? No right? So I would say lets keep it as simple as it was before and revert https://github.com/openshift/odo/pull/3835/files#diff-c127bd883f89dd3a7abff6183ad9921dL68 back. TBH its complicating the current spec and one more reason is, I can see only one spec in this file which verifies `sprinboot` devfile.yaml works well here - https://github.com/openshift/odo/pull/3835/files#diff-c127bd883f89dd3a7abff6183ad9921dL68 

Removing this spec doesn't make sense to me which verifies two thing:
- push works fine with `springboot devfile.yaml`
- We can also do `odo push` along with `name` field.

Reply:
but wouldn't this spec also verify its working of `springboot` and I have kept the non-component name push in this spec as well
https://github.com/openshift/odo/pull/3835/files#diff-c127bd883f89dd3a7abff6183ad9921dR92
So I am trying to understand why do you feel its complicating, its just consolidating a very small verification of push with name with other specs which seems like a good refactor to me.